style.css

```diff
@@ -0,0 +1,127 @@
+@import url('https://fonts.googleapis.com/css2?family=Anton&display=swap');
```

Comment:
This could also be done inside the HTML <head> tag
`<link href="https://fonts.googleapis.com/css2?family=Anton&display=swap" rel="stylesheet">`

Reply:
Yep, it could, but don't have to